### PR TITLE
[11.0] [MIG] base_optional_quick_create

### PR DESCRIPTION
**Pull request by pedrobaeza**
Follow-up of #13 with commit history optimized.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Already approved by @elicoidal

File: base_optional_quick_create/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Copyright 2013 Agile Business Group sagl (<http://www.agilebg.com>)
+# Copyright 2016 ACSONE SA/NA (<http://acsone.eu>)
+
+{
+    'name': "Optional quick create",
+    'version': '11.0.1.0.0',
+    'category': 'Tools',
+    'summary': "Avoid 'quick create' on m2o fields, on a 'by model' basis",
+    'author': "Agile Business Group,Odoo Community Association (OCA)",
+    'website': 'http://www.agilebg.com',
```
Comment (elicoidal) [inline base_optional_quick_create/__manifest__.py]: URL should be the target repo

**Review by elicoidal — APPROVED** [review]
LGTM

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM :+1:

File: base_optional_quick_create/__manifest__.py (hunk 1 above)
Comment (fmdl) [reply base_optional_quick_create/__manifest__.py]: @pedrobaeza con you update ?

Comment (pedrobaeza) [reply base_optional_quick_create/__manifest__.py]: Done

**Pull request merged**